RELEASE_NOTES.md

```diff
@@ -0,0 +1,69 @@
+# Release v0.11.0 #
+
+## New Features ##
+
+* Support team wise NFS storage, including:
+    - A NFS configuration plug-in and a commandline tool. [#2346](https://github.com/Microsoft/pai/pull/2346)
+    - A simple NFS-job submit plug-in. [#2358](https://github.com/Microsoft/pai/pull/2358)
+    
+* New alerts for unhealthy GPUs, currently including following alerts [#2209](https://github.com/Microsoft/pai/pull/2209): 
+    - gpu used by zombie container
+    - gpu used by external process
+    - gpu ecc error
+    - gpu hangs
+    - gpu memory leak
+  
+* Admin could know all running jobs on a node. [#2197](https://github.com/Microsoft/pai/pull/2197)
+    ![](./docs/release_note/images/node_tasks.png)
+
+* Filter supports in Job List View. [#302](https://github.com/Microsoft/pai/pull/302)
+    ![](./docs/release_note/images/filter.png)
+
+* Hold the Env for failed jobs which are casued by user error. [#2272](https://github.com/Microsoft/pai/pull/2272)
+
+
+## Improvements ##
+
+### Service ###
```

Comment:
Is `## ... ##` or `### ... ###` necessary?
`##` or `###` is enough for [markdown headers](https://help.github.com/en/articles/basic-writing-and-formatting-syntax#headings).

Reply:
Sure